java/compaction/compaction-job-creation/src/test/java/sleeper/compaction/job/creation/CreateCompactionJobsIT.java

```diff
@@ -107,16 +95,18 @@ void tearDown() {
     @Test
     public void shouldCompactAllFilesInSinglePartition() throws Exception {
         // Given
-        List<Partition> partitions = stateStore.getAllPartitions();
-        FileReferenceFactory fileReferenceFactory = FileReferenceFactory.from(partitions);
+        TableProperties tableProperties = createTable(schema);
+        StateStore stateStore = stateStoreProvider.getStateStore(tableProperties);
+        stateStore.initialise();
```

Comment:
It looks like the initialise call could be done inside createTable?

The TableProperties object doesn't seem to be used except to get the state store, so maybe createTable could just return the state store?